jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/BytecodeAnnotator.java

```diff
@@ -224,7 +224,9 @@ private static void copyAndAnnotateJarEntry(
       throws IOException {
     String entryName = jarEntry.getName();
     if (entryName.endsWith(".class")) {
-      jarOS.putNextEntry(new ZipEntry(jarEntry.getName()));
+      ZipEntry entry = new ZipEntry(jarEntry.getName());
+      entry.setTime(0);
+      jarOS.putNextEntry(entry);
```

Comment:
Can we extract these lines to a common private method and call it from all four places?  And, in that method, can we add a comment as to why we do `entry.setTime(0)`?